scripts/discover/as-root-domain-recon

```diff
@@ -23,11 +23,16 @@ declare -a dig_commands=("TXT" "NS" "MX" "SOA" "A")
 
 function root_domain_recon {
   domain=$(echo $1 | tr 'A-Z' 'a-z')
-  if ! echo "$domain" | as-prune-blacklisted-domains | grep "$domain" > /dev/null ; then
+  mkdir -p "recon/domains/$domain"
+
+  cat scope-domains*.txt 2>/dev/null \
+    | grep -P "$(echo "$domain" | sed 's/\./\\./g')\$" \
+    | sort -d \
+    | uniq > "recon/domains/$domain/subdomains-scope.txt"
+
+  if ! arsenic scope domains -r | grep -q "$domain"; then
```

Comment:
I think we are in the habit of doing things like this. I wonder if we should have a `scope check` command that returns a non zero status... Just a thought no need to change things here.